src/Raven.Client/Http/RavenCommand.cs

```diff
@@ -57,7 +57,7 @@ public abstract class RavenCommand<TResult>
 
         public RavenCommandResponseType ResponseType { get; protected set; }
 
-        public TimeSpan? Timeout { get; protected set; }
+        public TimeSpan? Timeout { get; internal set; }
```

Comment:
we cannot change that, probably you can use `protected internal`

Reply:
Done